hikey.mk

```diff
@@ -1,14 +1,12 @@
 ################################################################################
-# User-defined variables
-# Edit so these match your target
-# NOTE: If making changes after a build, please clean before rebuilding!
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
 ################################################################################
-# Non-secure user mode (root fs binaries): 32 or 64-bit [default 64]
-NSU ?= 64
-# Secure kernel (OP-TEE OS): 32 or 64-bit [default 64]
-SK ?= 64
-# Secure user mode (Trusted Apps): 32 or 64-bit [default 32, requires SK=64 for 64]
-SU ?= 32
+COMPILE_NS_USER   ?= 64
+COMPILE_NS_KERNEL := 64
```

Comment:
OK, I agree we don't want `?=` here. But `:=` won't force the value if it's provided as an argument to `make` (as opposed to via an environment variable). For this to happen you need `override COMPILE_NS_KERNEL := 64`.